lib/libbackscrub.h

```diff
@@ -0,0 +1,54 @@
+/* This is licenced software, @see LICENSE file.
+ * Authors - @see AUTHORS file.
+==============================================================================*/
+
+// tested against tensorflow lite v2.4.1 (static library)
+#ifndef _LIBBACKSCRUB_H
+#define _LIBBACKSCRUB_H
+
+// for cv::Mat and related types
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <stdarg.h>
+
+#define TFLITE_MINIMAL_CHECK(x)                              \
+  if (!(x)) {                                                \
+	fprintf(stderr, "Error at %s:%d\n", __FILE__, __LINE__); \
+	exit(1);                                                 \
+  }
```

Comment:
Consistent indentation

Reply:
I honestly don't like this macro at all - better to return appropriate codes from a library than ditch the whole program, I'll look at a better way of doing this.

Reply:
Removed entirely.